lib/Monocle_Alternate.php

```diff
@@ -235,8 +236,8 @@ public function query_gyms($conds, $params)
         r.move_1 AS raid_pokemon_move_1,
         r.move_2 AS raid_pokemon_move_2
         FROM forts f
-        LEFT JOIN fort_sightings fs ON fs.fort_id = f.id
-        LEFT JOIN raids r ON r.fort_id = f.id
+        LEFT JOIN fort_sightings fs ON (fs.fort_id = f.id AND fs.last_modified = (SELECT MAX(last_modified) FROM fort_sightings fs2 WHERE fs2.fort_id=f.id))
+        LEFT JOIN raids r ON (r.fort_id = f.id AND r.time_end >= extract(epoch from now()))
```

Comment:
I would recommend using a PHP function to get `time()` and pass that through as a parameter as `extract(epoch from now())` is not supported on MySQL, and conversely `UNIX_TIMESTAMP()` is not supported in Postgres